src/Softeq.NetKit.Notifications.Service/NotificationSenders/Sms/Validators/BaseSmsValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using FluentValidation;
+using Softeq.NetKit.Notifications.Service.NotificationSenders.Sms.Messages;
+
+namespace Softeq.NetKit.Notifications.Service.NotificationSenders.Sms.Validators
+{
+    public abstract class BaseSmsValidator<TModel> : AbstractValidator<TModel> where TModel : SmsMessage
+    {
+        protected BaseSmsValidator()
+        {
+            RuleFor(message => message.Text).NotEmpty();
```

Comment:
Add some limit for sms text as well as for message.Recipient

Reply:
done